__tests__/active-donors.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * This file is part of Serlo.org Cloudflare Worker.
+ *
+ * Copyright (c) 2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/serlo.org-cloudflare-worker for the canonical source repository
+ */
+import {
+  extractUserIdsFromSheet,
+  handleRequest,
+  isUserId,
+  Options,
+} from '../src/active-donors'
+import { createJsonResponse } from '../src/utils'
+import { mockFetch, FetchMock, expectHasOkStatus } from './_helper'
+
+describe('handleRequest()', () => {
+  const apiUrl = 'https://api.serlo.org/community/active-donors'
```

Comment:
Question: Shall we have the endpoint under `api.serlo.org` or better under `de.serlo.org`?

Reply:
Neither, should be part of the GraphQL API since we want all communication between services to go through that in the long-term.